script/nnbd_plugins.sh

```diff
@@ -7,6 +7,7 @@
 readonly NNBD_PLUGINS_LIST=(
   "android_intent"
   "battery"
+  "camera"
```

Comment:
I suggested this to fix the failing `build_all_plugins` test. `camera` has a conflict with `google_sign_in` while `camera_platform_interface` has a conflict with `google_maps_flutter`. I suggested that we keep them excluded from both until the camera plugin is migrated. 

The alternative to this would be to remove the camera plugin from the `build_all_plugins` test and run the camera tests on master. Would that be alright?

Reply:
Ahhh, I see! As this is right now, `build_all_plugins` is already not building the `camera` plugin, right? As long as some 'camera' tests run in some shard in `master`, it's probably enough!